turbinia/workers/analysis/windows_acct.py

```diff
@@ -58,8 +58,14 @@ def run(self, evidence, result):
     if num_files < 2:
       result.close(self, success=True, status='No Windows account files found')
       return result
-    (creds, hashnames) = self._extract_windows_hashes(
-        result, os.path.join(location, 'Windows', 'System32', 'config'))
+    try:
+      (creds, hashnames) = self._extract_windows_hashes(
+          result, os.path.join(location, 'Windows', 'System32', 'config'))
+    except TurbiniaException as e:
+      result.close(
+          self, success=False,
+          status='Unable to extract hashes from registry files: {0:s}'.format(
+              str(e)))
```

Comment:
I think we want to `return result` from here then too, right?

Reply:
Oh, yep!